src/js/providers/html5.js

```diff
@@ -629,7 +629,7 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
         _this.eventsOn_();
         // the loadeddata event determines the mediaType for HLS sources
         if (_levels.length && _levels[0].type !== 'hls') {
-            this.sendMediaType(_levels);
+            _setMediaType();
```

Comment:
There's no point it calling `_setMediaType()` here. We called `this.sendMediaType(_levels);` if we had levels. `init` is called before the video is loaded so we should not call methods that require the video tag to have loaded anything.

Reply:
The drawback here is for a setup with an audio file. Now we can't start loading the poster until the audio file is loaded.